pkg/katautils/config.go

```diff
@@ -596,12 +596,14 @@ func updateRuntimeConfigHypervisor(configPath string, tomlConf tomlConfig, confi
 }
 
 func updateRuntimeConfigProxy(configPath string, tomlConf tomlConfig, config *oci.RuntimeConfig, builtIn bool) error {
-	if builtIn {
-		config.ProxyType = vc.KataBuiltInProxyType
-		return nil
-	}
-
 	for k, proxy := range tomlConf.Proxy {
+		if builtIn {
+			config.ProxyType = vc.KataBuiltInProxyType
+			config.ProxyConfig = vc.ProxyConfig{
+				Debug: proxy.debug(),
```

Comment:
I'm not sure if it's correct to take the configuration of the first proxy that appears in the configuration file, since: 

1. This behaviour it's not documented. 
2. Proxy built-in is part of the runtime.

I'd prefer to use the runtime information instead.

```go
	Debug: config.Debug,
```

what do you think?

Reply:
Yeah, that's a good idea. 
Had updated!